server/config/db-connect.js

```diff
@@ -0,0 +1,30 @@
+//db-connect.js
+/* Created by Fil on August 25, 2019 */
+
+//TODO (FG) - Change require to import once Babel works?
+const config = require('./config.js');
+const mongoose = require('mongoose');
+
+const dbUrl = `${config.db.dbType}://${config.db.host}:${config.db.port}/${config.db.name}`;
+console.log (dbUrl);
```

Comment:
probably want to remove this console.log

Reply:
oops ... coming out now.